deploy/charts/harvester/values.yaml

```diff
@@ -457,16 +457,21 @@ harvester-load-balancer:
 kube-vip:
   enabled: false
   image:
-    repository: ghcr.io/kube-vip/kube-vip
-    tag: v0.5.12
+    repository: ghcr.io/kube-vip/kube-vip-iptables
+    tag: v0.6.0
```

Comment:
What are the differences between `ghcr.io/kube-vip/kube-vip` and `ghcr.io/kube-vip/kube-vip-iptables`? 
I have checked the images the `version` display Build and Version info are the same.
```

❯ docker run --rm ghcr.io/kube-vip/kube-vip-iptables:v0.6.0 version
Kube-VIP Release Information
Version:  v0.6.0
Build:    b594d459fc4e2061015157715fc84e84bee12d81
❯ docker run --rm ghcr.io/kube-vip/kube-vip:v0.6.0 version
Kube-VIP Release Information
Version:  v0.6.0
Build:    b594d459fc4e2061015157715fc84e84bee12d81
```

Reply:
The image ghcr.io/kube-vip/kube-vip-iptables is built from https://github.com/kube-vip/kube-vip/blob/main/Dockerfile_iptables, while ghcr.io/kube-vip/kube-vip from https://github.com/kube-vip/kube-vip/blob/main/Dockerfile. 

 ghcr.io/kube-vip/kube-vip-iptables: sha256:aab8a321a08726d2b9b49eedfdb23168f11adb55f0c21dcfbda3b48bc9af74bd
 ghcr.io/kube-vip/kube-vip: sha256:defef5b817487ca1f30123df5cfd17bad996d0b5f885e3004541810986d132d2